app/model/Upload.scala

```diff
@@ -0,0 +1,16 @@
+package model
+
+import org.cvogt.play.json.Jsonx
+import play.api.libs.json._
+
+case class UploadPolicy(bucket: String, key: String, region: String, credentials: UploadCredentials)
+case class UploadCredentials(temporaryAccessId: String, temporarySecretKey: String, sessionToken: String)
+
+object UploadPolicy {
+  implicit val format: Format[UploadPolicy] = Jsonx.formatCaseClass[UploadPolicy]
```

Comment:
I copied the existing pattern. Looking at the usages in the code, we could definitely just use `Json.format` built into Play here.

We do however use the `formatCaseClassWithDefaults` method from the library elsewhere. So I guess we then use the library everywhere for consistency?